eng/vs-dogfood.ps1

```diff
@@ -0,0 +1,31 @@
+function Dogfood-VS
+{
+    param(
+        [string]$vs = "${env:ProgramFiles}\Microsoft Visual Studio\2022\Preview\Common7\IDE\devenv.exe",
```

Comment:
Is it worth figuring out how to make this script work on Mac? I think it *might* work if we just set the right path for VS Mac and use `/` everywhere for directory separators.

Reply:
This does not appear to work on vsmac. Not sure if VSMAC checks the same env variables as windows.

Reply:
NM I got it working by passing in the environment variables